.github/workflows/batch_changelog.yml

```
@@ -0,0 +1,52 @@
+env:
+  GITHUB_TOKEN: ${{ secrets.BOT_TOKEN }}
+  CHANGELOG_DIR: ${{ vars.CHANGELOG_DIR }}
+
+jobs:
+  changelog:
+    runs-on: self-hosted
+    permissions:
+      contents: write
+    steps:
+      - name: Checkout Master
+        uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)